ganga/GangaCore/Utility/feedback_report.py

```diff
@@ -72,57 +71,47 @@ def get_content_type(filename):
         
         return data, file_dict
 
-    def make_upload_file(server):
-
-        def upload_file(path):
-
-            # print 'Uploading %r to %r' % (path, server)
-
-            data = {'MAX_FILE_SIZE': '3145728',
-                    'sub': '',
-                    'mode': 'regist'}
-            files = {'file': path}
-
-            return send_post(server, files)
-
-        return upload_file
+    def upload(filetype, filename, localdir):
+        try:
+                from GangaCore.GPIDev.Adapters.IGangaFile import IGangaFile
+                from GangaCore.GPIDev.Lib.File.LocalFile import LocalFile
+                from GangaCore.GPIDev.Lib.File.LCGSEFile import LCGSEFile
+                from GangaCore.GPIDev.Lib.File.GoogleFile import GoogleFile
+                from GangaCore.GPIDev.Lib.File.MassStorageFile import MassStorageFile
+                
+                filetype = stripProxy(filetype)
+                assert(issubclass(filetype, IGangaFile))
+                print(filename, localdir)
+                if issubclass(filetype, LocalFile):
```

Comment:
`LocalFIle` doesn't really need te execution of `put` as we point it to the exact location of the file. 
Another question, I get the following error when using the `put` function of LocalFile.
![image](https://user-images.githubusercontent.com/23381512/78851467-7ac8ee80-7a37-11ea-9379-2dc7fe826053.png)
Though the [`put`](https://github.com/ganga-devs/ganga/blob/d057b9e9f7fce0713190b4597c7e7ef8580608e3/ganga/GangaCore/GPIDev/Lib/File/LocalFile.py#L245) method seems to be implemented.

Reply:
Just posted #1633 to fix that problem. We can then avoid the if statement.

Reply:
Posted #1634 to fix this problem. I'll make the changes after its merger.